pkg/tfgen/generate_python.go

```diff
@@ -481,6 +482,7 @@ func (g *pythonGenerator) emitResourceType(mod *module, res *resourceType) (stri
 
 	// Old versions of TFGen emitted parameters named __name__ and __opts__. In order to preserve backwards
 	// compatibility, we still emit them, but we don't emit documentation for them.
+	w.Writefmt(", __props__=None")
```

Comment:
Do we actually need `__props__`? I thought we kind of figured out that this was a mistake in Node...

Reply:
High order bit for now: parity with Nodejs.  Decision about removing props from being used in reads is punted to a later point when we can make the decision for both.